Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -99,11 +128,13 @@ class VDPortGroup : BaseDSC {
         }
         finally {
             $this.DisconnectVIServer()
+            Write-VerboseLog -Message $this.SetMethodEndMessage -Arguments @($this.DscResourceName)
         }
     }
 
     [bool] Test() {
         try {
+            Write-VerboseLog -Message $this.TestMethodStartMessage -Arguments @($this.DscResourceName)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.